src/client/driver/command-executors/execute-pending-browser-manipulation.js

```diff
@@ -0,0 +1,50 @@
+import hammerhead from '../deps/hammerhead';
+import testCafeCore from '../deps/testcafe-core';
+import MESSAGE from '../../../test-run/client-messages';
+import DriverStatus from '../status';
+
+var nativeMethods = hammerhead.nativeMethods;
+var transport     = hammerhead.transport;
+var delay         = testCafeCore.delay;
+
+const CHECK_TITLE_INTERVAL           = 50;
+const APPLY_DOCUMENT_TITLE_TIMEOUT   = 500;
+const RESTORE_DOCUMENT_TITLE_TIMEOUT = 100;
+
+
+export default function executePendingBrowserManipulation () {
+    var savedDocumentTitle   = document.title;
+    var assignedTitle        = `[ ${window.location.toString()} ]`;
+    var checkTitleIntervalId = null;
+    var executionError       = null;
+
+    // NOTE: we should keep the page url in document.title
+    // while resizing is executing
```

Comment:
If somehow this PR is ready for merge before the Resize command... You know...
https://github.com/DevExpress/testcafe/pull/543#discussion_r63851226